### PR TITLE
Remove unnecessary >> $BASH_ENV

### PR DESCRIPTION
**Pull request by ataylorme**

### CODE REVIEW

File: scripts/set-environment [hunk 1]
```diff
@@ -78,8 +79,8 @@ TERMINUS_ENV=${TERMINUS_ENV:-$DEFAULT_ENV}
   echo 'export BEHAT_ADMIN_EMAIL=no-reply+ci-$CI_BUILD_NUMBER@getpantheon.com'
   echo "export MULTIDEV_SITE_URL='https://$TERMINUS_ENV-$TERMINUS_SITE.pantheonsite.io/'"
   echo "export LIVE_SITE_URL='https://live-$TERMINUS_SITE.pantheonsite.io/'"
-  echo "export ARTIFACTS_DIR='artifacts'" >> $BASH_ENV
```
Comment (ataylorme) [inline scripts/set-environment:81]: Not sure why these have been working with the extra `>> $BASH_ENV`

File: scripts/set-environment [hunk 2]
```diff
@@ -61,6 +61,7 @@ TERMINUS_ENV=${TERMINUS_ENV:-$DEFAULT_ENV}
 #=====================================================================================================================
 (
   echo 'export PATH=$PATH:$HOME/bin'
+  echo "export PATH='$HOME/.composer/vendor/bin:$PATH'"
```
Comment (ataylorme) [inline scripts/set-environment]: I'm not sure if this can run here or if it needs to run before `composer global require`

Comment (greg-1-anderson) [reply scripts/set-environment]: DON'T USE COMPOSER GLOBAL REQUIRE!

**Comment by ataylorme** [comment]
So Composer Lock Updater and HUB were already installed correctly...

**Review by greg-1-anderson — APPROVED** [review]

**Pull request merged**